scripts/bootstrap-dev.sh

```diff
@@ -7,42 +7,38 @@ export LC_ALL=en_US.UTF-8
 export LANG=en_US.UTF-8
 export PATH="/usr/bin:/bin:/usr/sbin:/sbin:$PATH"
 
-export VERSION_FLAG=https://cdn.asahilinux.org/installer-dev/latest
-export INSTALLER_BASE=https://cdn.asahilinux.org/installer-dev
-export INSTALLER_DATA=https://github.com/AsahiLinux/asahi-installer/raw/main/data/installer_data.json
 export REPO_BASE=https://cdn.asahilinux.org
+export INSTALLER_BASE="${REPO_BASE}"/installer-dev
+export VERSION_FLAG="${INSTALLER_BASE}"/latest
+export INSTALLER_DATA=https://github.com/AsahiLinux/asahi-installer/raw/main/data/installer_data.json
 
 #TMP="$(mktemp -d)"
 TMP=/tmp/asahi-install
 
 echo
-echo "Bootstrapping installer:"
+echo "  Bootstrapping installer:"
 
 mkdir -p "$TMP"
 cd "$TMP"
 
 echo "  Checking version..."
-
 PKG_VER="$(curl --no-progress-meter -L "$VERSION_FLAG")"
 echo "  Version: $PKG_VER"
-
 PKG="installer-$PKG_VER.tar.gz"
 
 echo "  Downloading..."
-
 curl --no-progress-meter -L -o "$PKG" "$INSTALLER_BASE/$PKG"
 curl --no-progress-meter -L -O "$INSTALLER_DATA"
 
 echo "  Extracting..."
-
 tar xf "$PKG"
 
 echo "  Initializing..."
 echo
 
 if [ "$USER" != "root" ]; then
-    echo "The installer needs to run as root."
-    echo "Please enter your sudo password if prompted."
+    echo "  The installer needs to run as root."
```

Comment:
Same here, this is a message and not part of the bootstrap so it is not supposed to be indented.